### ISSUE TITLE
Pix2Pix tutorial error: "Cannot convert a symbolic Tensor (truediv:0) to a numpy array."

### ISSUE THREAD

**Issue by lcdavis13** [tensorflow/tensorflow#36768]
## URL(s) with the issue: 

https://www.tensorflow.org/tutorials/generative/pix2pix#load_the_dataset

## Description of issue (what needs changing):

plt.imshow(inp/255.0) is not valid in latest TF. Instead you must call .eval() on the Tensor objects to convert them to numpy.

### Clear description

The existing code throws this error: "Cannot convert a symbolic Tensor (truediv:0) to a numpy array."
from this line: plt.imshow(inp/255.0)

I was able to fix this by calling inp.eval(). But in order to call eval, you also must set up a session, which has not yet been explained at that point in the tutorial.

There may be a better way to get around this error -- I am new to TensorFlow. 
I'm guessing the tutorial contains legacy code from 1.x ? I hope there isn't more because it took me quite a while to find a fix!

**Comment by gadagashwini** [comment]
@lcdavis13, I tried with Tf 2.1 and Latest Tf nightly version, its working as expected no error seen. Thanks

**Comment by lcdavis13** [comment]
Hi, thanks for the response. It seems to work as expected if I add the line "tf.enable_eager_execution()" , which is not included in the tutorial. But it then throws a warning about this method being deprecated. I've tested on 3 different environments and all show the same issue when following the tutorial. I'm not using the nightly builds though, but rather the latest Anaconda releases. Tested on one linux (redhat) and two Windows 10 environments.
